tests/conftest.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 from adbutils import adb, AdbDevice
+import subprocess
```

Comment:
subprocess is useless

Reply:
Done in [2fa8604](https://github.com/openatx/adbutils/pull/100/commits/2fa8604b4625e8c4b6b97cc968c91e66595d7514). I also removed the `pathlib` import as it is not needed either.